src/main/java/org/roda_project/commons_ip2/validator/component/descriptiveMetadataComponent/DescriptiveMetadataComponentValidator.java

```diff
@@ -561,7 +562,7 @@ private ReporterDetails validateCSIP24(final StructureValidatorState structureVa
     for (MdSecType mdSec : dmdSec) {
       final MdSecType.MdRef mdRef = mdSec.getMdRef();
       final String href = mdRef.getHref();
-      if (href != null) {
+      if (href != null && StringUtils.isNotBlank(href)) {
```

Comment:
As stated in the corresponding issue these are different scenarios which should be reflected in the error message. This way you'd get a "can't  be null" error for an empty attribute, which would be misleading.